src/EditorFeatures/TestUtilities/MoveToNamespace/TestMoveToNamespaceOptionsService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+extern alias WORKSPACES;
+
+using System;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.MoveToNamespace;
+using WORKSPACES::Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.MoveToNamespace
+{
+    [Export(typeof(IMoveToNamespaceOptionsService))]
+    [PartNotDiscoverable]
+    class TestMoveToNamespaceOptionsService : IMoveToNamespaceOptionsService
```

Comment:
Can you add tests for varying namespace names (with/without dots, etc.) and tests for when the operation is cancelled?